_episodes/05-contribution-governance-communication.md

```diff
@@ -1,52 +1,380 @@
+the community to contribute to its development. With their involvement,
+developers can improve their skills and connect people with similar
+interests while building their profile as an active contributor in the project.
+
+### Welcome your contributors!
+
+Make your contributors feel welcome, treat them well, respond to their queries
+and allow them to demonstrate their abilities. Being welcoming to your contributors
+sets the tone for future contributors, and incites future  contributors to
+also contribute to your project. To allow meaningful contributions,
+it is crucial to make your project [FAIR](https://www.nature.com/articles/sdata201618)
+(Findable, Accessible, Interoperable and Reusable). Define your project clearly,
+by listing the aim of the project, areas of applications and area of improvements.
+List all the availabilities such as your source-code, datasets, metadata,
+documentation, list of tasks, and contribution guideline.
+
+### Respect your contributors!
+
+It is a lot to consider, but remember that in the vast majority of cases,
+people take time to contribute to your project because they find what you created
+useful and want to improve it. These contributors may spend considerable amount of
+their time (often their unpaid, free time) to work on your project, so be respectful.
+If your issue tracker is filled with ignored emails or comments, rude replies,
+and uncalled bad behavior, your project will not be attractive for the developers.
+
+> ## Discussion: What should others know before contributing to your project?
+> #### Time: 10 min
+> Have you ever contributed to a project? What did you need to know to
+> contribute to that project? Based on your own experience, list the minimum
+> required information someone might require to contribute to your project.
+>
+> > ## Outcome
+> >
+> > In this exercise, learners should realise the importance of providing appropriate
+> > information to allow others to make contributions to your project without
+> > overwhelming or confusing them.
+> >
+> > Then you might suggest to include a minimum example document for contributing.
+> > The contribution file should contain the bare minimum information:
+> > - location: root-project/CONTRIBUTING.md
+> > - cheer: Give a super nice welcome to contributors!
+> > - introduction: Overview of the document
+> > - how-to: Cover one or two examples of contributions and time frame
+> >
+> > References:
+> > - https://rubygarage.org/blog/how-contribute-to-open-source-projects
+> {: .solution}
+{: .discussion}
+
+If you are not yet convinced that having your project open for contributions is
+a great idea, we can go through some questions that potential
+contributors to your open source project might want you to consider.
+
+1. Where can I contribute?
+2. How can I contribute?
+3. Who should I contact?
+4. What are the communication channels?
+5. What else should I know?
+6. What's in it for me?
+
```

Comment:
I think there is a gap here; from the 6 questions to the single question, seems like an abrupt change. Maybe add a sentence in the form of "_Although addressing all these questions is a very good idea, sometimes people just want to focus on a single thing_".

Reply:
great suggestion added and rephrased